helpers/files-storage.js

```diff
@@ -32,14 +32,19 @@ const getThumbnailIcon = (filename) => {
 	return thumbnail;
 };
 
-async function filesStorageInit(schoolId, parentId, parentType, req, readonly = false) {
+async function filesStorageInit(schoolId, next, parentId, parentType, req, readonly = false) {
```

Comment:
Die Reihenfolge sieht komisch aus, ich glaube so währe angemessen. Zuerst die props die für Funktion wichtig sind und dann die Technische props.

```suggestion
async function filesStorageInit(schoolId, parentId, parentType, readonly = false, req, next) {
```